examples/kmm-sample/build.gradle.kts

```diff
@@ -1,5 +1,8 @@
 buildscript {
     repositories {
+        if (Realm.ciBuild) {
```

Comment:
Can remember if we are allowed to have logic before the `buildScript` section, but maybe define one variable for the configuration and initialize it with `Realm.ciBuild`. 
```
val addBuildFolderRepository = Realm.ciBuild
```

Reply:
it isn't possible to have a global variable, but I could configure the extra properties instead.